Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM node:lts-slim AS frontend
+
+WORKDIR /app
+
+COPY package.json /app/
+RUN npm install
+COPY . /app/
+RUN npm run build
+
+FROM python:3.7-slim
```

Comment:
If we add the `AS` I think we can then have a docker compose file with a `target` for local development of the backend.